app/services/reject_application.rb

```diff
@@ -1,18 +1,24 @@
 class RejectApplication
+  attr_reader :rejection_reason
+
   include ActiveModel::Validations
 
-  def initialize(application_choice:, rejection:)
+  validates_presence_of :rejection_reason
+  validates_length_of :rejection_reason, maximum: 255
```

Comment:
Interesting, I'm using `validates:` in my parallel PR for the Personal details validations, e.g.:

```ruby
    validates :first_name, :last_name, :english_main_language,
              :first_nationality,
              presence: true

    validates :first_name, :last_name,
              length: { maximum: 100 }
```

I'm not sure if my branch passes Rubocop yet (have some trouble running the bugger locally), but do we want consistency on this? Tbh I'm fine with mixing the two kinds of sugar in our codebase, but if we want 1 flavour I'm happy to switch to this vice/versa etc

Reply:
No strong feelings — I think I prefer the `validates` version as it naturally groups rules by field. happy to conform

Reply:
No strong feelings either so feel free to leave as is, it's all readable to me 👍